eotsmanager/eotsmanager.go

```diff
@@ -28,12 +28,11 @@ type EOTSManager interface {
 	UnsafeSignEOTS(uid []byte, chainID []byte, msg []byte, height uint64) (*btcec.ModNScalar, error)
 
 	// SignSchnorrSig signs a Schnorr signature using the private key of the finality provider
-	// It fails if the finality provider does not exist or the message size is not 32 bytes
-	// or passPhrase is incorrect.
+	// It fails if the finality provider does not exist or the message size is not 32 bytes.
 	// uid represents EOTS key
 	SignSchnorrSig(uid []byte, msg []byte) (*schnorr.Signature, error)
 
-	// Unlock makes the private key for the given EOTS key (uid) accessible in memory using the provided passphrase.
+	// Unlock makes the private key for the given EOTS key (uid) accessible in memory using the provided keyring passphrase.
```

Comment:
```suggestion
	// Unlock makes the private key for the given EOTS key (uid) accessible in memory using the provided keyring key password.
```

Should we rename to password? I believe it is called password by the keyring file in cosmos https://github.com/cosmos/keyring/blob/master/file.go#L31